app/antedecorators/fee/fixed_gas.go

```diff
@@ -0,0 +1,54 @@
+package fee
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+	"github.com/NibiruChain/nibiru/app/antedecorators/types"
+	oracletypes "github.com/NibiruChain/nibiru/x/oracle/types"
+)
+
+const OracleMessageGas = 500
```

Comment:
Why 500 specifically?

Reply:
We did some back-of-the-envelope calculations. We deduced that this amount of `unibi` would allow oracles to post prices for approximately 3 weeks without running out of gas, using the 100 NIBI obtainable from the faucet.